inception-recommendation/pom.xml

```diff
@@ -229,6 +229,18 @@
         </exclusion>
       </exclusions>
     </dependency>
+    <dependency>
```

Comment:
The module must not hard-depend on any recommenders.  See comment further down regarding instantiation of recommenders.

Reply:
Fixed.